eHarmony/aloha
# Closes #186

## Summary
Requires `tut` documentation to compile as a build step.  Minimal fix for current broken docs.

## Resolves
* Fixes issue #186 

## How to Verify
Check the travis logs to make sure `docs:tut`  is run.  If so, this should be good to go.

### Code Reviewer(s)
@jmorra
